indico/htdocs/js/indico/jquery/global.js

```diff
@@ -61,8 +61,15 @@ $(document).ready(function() {
         var extraOpts = $(this).data('qtipOpts') || {},
             qtipClass = $(this).data('qtip-style');
 
-        $(this).qtip($.extend(true, {}, {
+        var target = $(this);
```

Comment:
"usually"? mostly for `$this`. it's pretty much a matter of personal style though. i don't like it that much, i think @pferreir  usually does it though